### ISSUE TITLE
Failed to setup CNI bridge during python example startup

### ISSUE THREAD

**Issue by arborum-x** [tilt-dev/tilt#5929]
### Expected Behavior
Example python pod (template from https://github.com/tilt-dev/tilt-example-python) running and port available for connections from localhost.

### Current Behavior
Pod stuck in status ContainerCreating with error message:
`failed to set bridge addr: could not add IP address to "cni0": permission denied`

#### `tilt doctor` Output

```
$ tilt doctor
...

Tilt: v0.30.7, built 2022-08-12
System: linux-amd64
---
Docker
- Host: unix:///var/run/docker.sock
- Server Version: 20.10.17
- API Version: 1.41
- Builder: 2
- Compose Version: v2.6.0
---
Kubernetes
- Env: minikube
- Context: minikube
- Cluster Name: minikube
- Namespace: default
- Container Runtime: containerd
- Version: v1.24.3
- Cluster Local Registry: &RegistryHosting{Host:localhost:44371,HostFromClusterNetwork:ctlptl-registry:5000,HostFromContainerRuntime:ctlptl-registry:5000,Help:https://github.com/tilt-dev/ctlptl,SingleName:,}
---

```

**Comment by nicks** [comment]
This error message means your cluster is broken. Any pod you deploy will be broken, not just ones deployed by tilt. 

We usually recommend people use tilt with kind or docker-desktop because they have far fewer footguns. See this guide: https://docs.tilt.dev/choosing_clusters.html

Or, if you want help diagnosing why your minikube is broken, I recommend the minikube docs. https://minikube.sigs.k8s.io/docs/

**Comment by arborum-x** [comment]
Ah ok, I decided to try out the [ctlplt](https://github.com/tilt-dev/ctlptl#minikube-with-a-built-in-registry) route with minikube, which apparently does not yet work well together out of the box with the python example (using `ctlptl create cluster minikube --registry=ctlptl-registry`). I then tried using the standard `minikube start` command and that managed to start the pod correctly. 

Thank you for the pointers. Perhaps the documentation could be updated to prefer a standard minikube startup instead of the ctlplt way until it works nicely out of the box together with the [examples](https://github.com/tilt-dev/tilt-example-python) from the docs.
